ansible_collections/arista/avd/plugins/plugin_utils/cv_client/workflows/deploy_cv_pathfinder_metadata_to_cv.py

```diff
@@ -6,6 +6,7 @@
 from copy import deepcopy
 from logging import getLogger
 
+from ...password_utils.password import simple_7_decrypt
```

Comment:
do we need to decrypt? Can't this be down on CV side?

Reply:
`Can't this be down on CV side?` No.
We will move to using CV "SecretService" in a later stage.